Automattic/woocommerce-payments
# Instant Deposit button and modal design tweaks

Fixes #1666 

#### Changes proposed in this Pull Request

* ~Makes the Instant Deposit button visible all the time, but disabled if it cannot be used. Due to this, it would not be good to push this live until the server PR is done and docs are created.~
* Button is only shown if merchant is enabled for instant deposits, will show disabled if requirements for deposit are not met at the time of viewing.
* Removed Close button from modal due to there is an X to close the modal, and if you click off of it it closes.
* Updated fee percentage to match setting on server via value returned from API call.
* Update tests and snapshots accordingly.

#### Testing instructions

1. Use the trunk server branch.
2. When viewing Payments > Deposits the Instant Deposit button should not be visible.

---

1. Use a server branch that has instant deposits available, the branch in 455-gh-Automattic/woocommerce-payments-server is the best to use, unless it has been merged.
2. Perform a transaction greater than $50 through your store with card number `4000000000000077`, which is [found in the Stripe docs](https://stripe.com/docs/testing#cards-responses).
3. When viewing Payments > Deposits in the admin, the Instant Deposit button should appear.
4. Click the button to open the modal, the percentage amounts should be 1.5%.
5. The Close button should now be gone.

-------------------
